fix the bug #171

remybaranx opened the pull request:
Now the `getSelectedValue()` method supports AbstractButton components such as JCheckBox.
I have added a new "GetSelectedText" test case in the playback demo, to validate this modification.

lvboque commented:
For code review

dergo commented:
I don't agree with your changes: getSelectedValue() should return the selected value, not the label of the button, which is done by the getText() method !
For a check box, it should return True if the check box is selected, False otherwise.

I think GetSelectedText test should be removed and Selection test should be completed.

remybaranx commented:
Yes, you're right, I misunderstand this bug.

- I have replaced AbstractButton.getText() by AbstractButton.isSelected().
- I have removed the GetSelectedText test
- I have updated the Selection test

(You can see these modifications directly in this pull request)
